### PR TITLE
#181 Add template helper

### PR DESCRIPTION
**Pull request by tuxBurner**
Hi @leleuj,
here is the promised PR for issue #181

I also made an example: https://github.com/tuxBurner/play-pac4j-scala-demo/tree/issue/%23181

Greetings tuxburner

### CODE REVIEW

File: shared/src/main/scala/org/pac4j/play/scala/Pac4jScalaTemplateHelper.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  *
+  * In your Action the request value must be declared implicit.
+  *
+  * In your templates add this as implicit constructor values: (implicit pac4jScalaTemplateHelper: Pac4jScalaTemplateHelper[CommonProfile], requestHeader: RequestHeader)
+  *
+  * For an example take a look in the play-pac4j-scala-demo: ApplicationWithScalaHelper
+  *
+  *
+  * @author Sebastian Hardt
+  */
```
Comment (leleuj) [inline shared/src/main/scala/org/pac4j/play/scala/Pac4jScalaTemplateHelper.scala]: Can you add an `@since 6.0.0` to track the new class?

Comment (tuxBurner) [reply shared/src/main/scala/org/pac4j/play/scala/Pac4jScalaTemplateHelper.scala]: Yes of course

**Comment by leleuj** [comment]
Excellent! One more thing and we're good to merge: can you add a few lines for this in the documentation (in this section: https://github.com/pac4j/play-pac4j#5-get-the-user-profile-profilemanager)? Thx

**Comment by leleuj** [comment]
Perfect! Merging

**Pull request merged**